spec/might/filter_parameters_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'might/filter_parameters'
+require 'might/filter_parameter_definition'
+require 'might/filter_parameter'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.